frontend/src/components/App/App.jsx

```diff
@@ -1,25 +1,26 @@
-import './App.css'
-import Navbar from '../Navbar/Navbar'
-import Search from '../Search/Search'
-import { useState } from 'react';
+import "./App.css";
+import Navbar from "../Navbar/Navbar";
+import Search from "../Search/Search";
+import { useState } from "react";
 
 export default function App() {
-  const [searchInput, setSearchInput] = useState("");
-  const [currentStock, setCurrentStock] = useState("");
+    const [searchInput, setSearchInput] = useState("");
+    const [currentStock, setCurrentStock] = useState("");
```

Comment:
It seems like this is not the stock itself, but the stock ticker. 
There's also not guarantee that the search input is a valid ticker.